src/styles/partials/custom.scss

```diff
@@ -0,0 +1,18 @@
+// Define a color variable
+$color-sassy: #cc4397;
+$color-blue: #0059ff;
+
```

Comment:
Good job Dami, I noticed that there is a file named variables.scss for declaring all variables and here too in custom.scss, you declared those same variables. Can we have the variable declarations only in the files where they are needed or is there another use for the variables file?

Reply:
The purpose for the variables file is to have general styles, and call the variables when needed, anytime we need the variables in our component we import the variable file into that component. As for these lines, you pointed out I didn't remove these lines while I was refactoring into other files.
thank you.

Reply:
Okay..cool